app/src/validators/team.validator.js

```diff
@@ -11,8 +11,9 @@ class TeamValidator {
         ctx.checkBody('layers').optional();
         const isArrayToErrors = (field) => {
           const value = ctx.request.body[field];
-          if (typeof value !== 'undefined' && !Array.isArray(value)) {
-            ctx.errors = [ ...ctx.errors , { [field]: `${field} should be an Array.` }];
+          if (value !== undefined && !Array.isArray(value)) {
```

Comment:
I have changed it but I think comparing with a string could be also a risk, even if its minimal